src/loader.c

```diff
@@ -1102,9 +1189,12 @@ struct db_main *ldr_init_test_db(struct fmt_main *format, struct db_main *real)
 
 	bench_running++;
 	while (current->ciphertext) {
-		char line[LINE_BUFFER_SIZE];
+		char *ex_len_line = NULL, _line[LINE_BUFFER_SIZE], *line = _line;
 		int i, pos = 0;
-
+		if (strlen(current->ciphertext) > LINE_BUFFER_SIZE-200) {
+			ex_len_line = mem_alloc(strlen(current->ciphertext)+300);
+			line = ex_len_line;
+		}
```

Comment:
What are "200" and "300" here? Should we derive them from some macro constants instead?